tests/test_FFBbasis2D.py

```diff
@@ -121,8 +115,6 @@ def testRotate(self):
         basis = FFBBasis2D((x1.res,) * 2, dtype=x1.dtype)
```

Comment:
can instantiate with just int size, easier on the eyes

Reply:
changed